server/database/config/connection.js

```diff
@@ -1,4 +1,6 @@
 const { Pool } = require('pg');
+require('env2')('config.env');
```

Comment:
you should require this line one time in app

Reply:
Done!